datasets/cmu_mocap/README.md

```diff
@@ -0,0 +1,169 @@
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** [CMU MoCap homepage](http://mocap.cs.cmu.edu/)
+- **Repository:** [More Information Needed]
+- **Paper:** [More Information Needed]
+- **Leaderboard:** [More Information Needed]
+- **Point of Contact:** [CMU MoCap Contact](mailto:jkh+mocap@cs.cmu.edu)
+
+### Dataset Summary
+
+The CMU Graphics Lab Motion Capture Database is a dataset containing motion capture recordings of people performing different actions such as walking, jumping, dancing etc. The actions have been performed by over 140 subjects. There are a total of 2605 trials in 6 categories and 23 subcategories.
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+All the categories, sub-categories and descriptions use English as their primary language.
+
+## Dataset Structure
+
+### Data Instances
+
+A sample from the CMU MoCap dataset is provided below:
+```
+{
+    'subject_id': 12, 
+    'categories': ['Physical Activities & Sports', 'Locomotion', 'Locomotion', 'Locomotion'], 
+    'subcategories': ['martial arts', 'walking', 'walking', 'walking'], 
+    'descriptions': ['tai chi', 'walk', 'walk', 'walk'], 
+    'amc_files': ['~/all_asfamc/subjects/12/12_04.amc', '~/all_asfamc/subjects/12/12_01.amc', '~/all_asfamc/subjects/12/12_03.amc', '~/all_asfamc/subjects/12/12_02.amc'], 
+    'asf_file': '~/all_asfamc/subjects/12/12.asf',
+}
+```
+Depending on the file format, 'files' field would change. Check the Data Fields section for more information.
+
+### Data Fields
+
+The following fields are common for all file formats of the dataset.
+
+- `subject_id`: Unique ID for each subject in the MoCap dataset.
+- `categories`:  A list containing the category names of all trials.
+- `subcategories`: A list containing the subcategory names of all trials.
+- `descriptions`: A list containing the descriptions of all trials.
+
+According to the file format, we have the following fields:
+- `amc_files`, `c3d_files`, `mpg_files` , `avi_files` : A list containing the path to all files.
+
+The "asf/amc" config has an additional field :
+
+- `asf_file`: ASF file is the skeleton file, in the ASF file a base pose is defined for the skeleton that is the starting point for the motion data. More info can be found [here](https://research.cs.wisc.edu/graphics/Courses/cs-838-1999/Jeff/ASF-AMC.html).
```

Comment:
Let's expose the other formats via configs rather than via a parameter. Also, let's rename `{format}_files` to `motions` (for the `asf/amc` config this field can be a dictionary).

Reply:
Yes, I'll change {format}_files to motions.
Also, may I know why using configs instead of parameter would be a better option ? I'll work on this part.

Reply:
Parameters are more hidden, and we need an extra section to document them. And configs are shown in the viewer and an error message when the user specifies an unsupported config.

Reply:
Added the formats as configs and renamed {format}_files to motions. Updated the readme accordinly.